i18n/en-US.properties

```diff
@@ -1437,9 +1437,11 @@ boxui.unifiedShare.collaborators.expirationTooltipClickableText = Access expires
 # Text to show when the number of contact email addresses displayed on a tooltip exceeds the maximum amount that can be displayed
 boxui.unifiedShare.contactEmailsTooltipText = {emails}, and {remainingEmailsCount} more
 # Text for the notice that is displayed when there are collaboration restrictions that apply to one or more of the selected contacts
-boxui.unifiedShare.contactRestrictionNotice = Invitations cannot be sent to {count, plural, one {{email}} other {{count} people}} because external collaboration is restricted due to the applied classification.
+boxui.unifiedShare.contactRestrictionNotice = Invitations cannot be sent to {count, plural, one {{count} person} other {{count} people}} because external collaboration is restricted due to the applied classification.
```

Comment:
This message won't actually be used for the single contact scenario, but need to keep the `one` definition for correctness.